lib/capistrano/tasks/systemd.rake

```diff
@@ -58,8 +58,8 @@ namespace :sidekiq do
         git_plugin.execute_systemd('enable', git_plugin.sidekiq_service_file_name(config_file))
       end
 
-      if fetch(:sidekiq_enable_lingering) && fetch(:sidekiq_lingering_user)
```

Comment:
Linger is not enabled if `:sidekiq_lingering_user` is empty. I think it's not expected because `loginctl enable-linger` by default will enable linger for current user (which capistrano-puma does)